common/alert/reports/a11y-1.json

```
@@ -0,0 +1,847 @@
+{
+  "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+  "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": "https://wet-boew.github.io/GCWeb/common/alert/en.html",
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "",
+    "earl:pointer": {
+      "@type": "",
+      "@value": ""
+    }
+  },
+
+  "earl:assertedBy": {
+    "foaf:name": "Service Canada - Principal Publisher",
+    "foaf:homepage": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "earl:mainAssertor": {
+      "foaf:name": "Service Canada - Principal Publisher",
+      "foaf:homepage": "https://github.com/Shiva-sc",
+      "@type": ["earl:Assertor", "foaf:Person"],
+      "foaf:member": {
+        "@id": "_:accessibility_insights",
+        "doap:name": "Accessibility Insights for Web",
+        "doap:homepage": "https://accessibilityinsights.io/",
+        "doap:repository": "https://github.com/microsoft/accessibility-insights-web",
+        "doap:created": "2022-09-30",
+        "doap:release": "v2.35.0",
+        "@type": ["earl:Assertor", "earl:Software"]
+      }
+    }
```

Remove the foaf:member considering you didn't used any automated tool.

```suggestion
      "@type": ["earl:Assertor", "foaf:Person"]
    }
```